File: 1/p1.py
#!/bin/env python3

L_RANGE = 1
R_RANGE = 1000

sum = 0

for num in range(L_RANGE, R_RANGE):
    if (num % 3 == 0) or (num % 5 == 0):
        sum += num

print("sum = ", sum)

File: 1/p1.c
#include <stdio.h>

int main()
{
    unsigned int sum = 0, i = 1;
    while(i < 1000)
    {
        if(i%3 == 0 || i%5 == 0)
        {
            sum += i;
        }
        i++;
    }
    printf("sum = %d\n", sum);
    return 0;
}
